addon/components/acl-wizzard.hbs

```diff
@@ -0,0 +1,104 @@
+{{#if this.modelToSelect}}
+  <div class="uk-margin">
+    <SectionTitle @model="{{this.modelToSelect}}s" @noCreateLink="true" />
+    <DataTable @modelName={{this.modelToSelect}} as |table|>
+      <table.head>
+        {{#each (get this.fields this.modelToSelect) as |field|}}
+          <th>
+            {{t (concat "emeis." this.modelToSelect "s.headings." field)}}
+          </th>
+        {{/each}}
+      </table.head>
+      <table.body as |body|>
+        <body.row
+          class="pointer"
+          data-test-row
+          {{on "click" (fn this.selectModel body.model)}}
+        >
+          {{#with body.model as |model|}}
```

Comment:
Use `{{let}}`

Reply:
https://github.com/emberjs/rfcs/blob/master/text/0286-block-let-template-helper.md#deprecating-with

Reply:
Also, if it was falsy you would have other problems in the lines above.